midx.c

```diff
@@ -800,8 +804,8 @@ static int write_midx_internal(const char *object_dir, struct multi_pack_index *
 						 packs_to_drop->items[drop_index].string);
 
 				if (!cmp) {
+					expire_int_ids[drop_index] = i;
```

Comment:
GH won't let me make a comment on a non delta line, so this is the closest line.

https://github.com/Microsoft/git/pull/106/files#diff-4d791aa648626485aa1adb43d8972a33R812

On line 812 below, incrementing drop_index will leave a zero cell in expire_int_ids[].
Can't tell yet if that is a good thing or a bad thing.